Src/FluentAssertions.Json/JTokenAssertions.cs

```diff
@@ -187,6 +187,11 @@ public AndConstraint<JTokenAssertions> HaveValue(string expected)
         /// </param>
         public AndConstraint<JTokenAssertions> HaveValue(string expected, string because, params object[] becauseArgs)
         {
+            Execute.Assertion
+                .ForCondition(Subject is object)
+                .BecauseOf(because, becauseArgs)
+                .FailWith("Expected JSON token to have value {0}, but the element was <null>.n", expected); 
```

Comment:
❓ Should `expected` be called with `expected.EscapePlaceholders()` as other places in this file?

Reply:
No, only when it is used directly in the formatting string.